definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -787,41 +782,41 @@ declare module "lodash" {
     after(n: number, fn: Function): Function;
     ary(func: Function, n?: number): Function;
     before(n: number, fn: Function): Function;
-    bind<F:(...any[]) => any>(func: F, thisArg: any, ...partials: Array<any>): F;
+    bind<F:(...Array<any>) => any>(func: F, thisArg: any, ...partials: Array<any>): F;
     bindKey(obj?: ?Object, key?: ?string, ...partials?: Array<?any>): Function;
     curry: Curry;
     curry(func: Function, arity?: number): Function;
     curryRight(func: Function, arity?: number): Function;
-    debounce<F: (...any[]) => any>(
+    debounce<F: (...Array<any>) => any>(
       func: F,
       wait?: number,
       options?: DebounceOptions
     ): F & Cancelable;
-    defer(func: (...any[]) => any, ...args?: Array<any>): TimeoutID;
+    defer(func: (...Array<any>) => any, ...args?: Array<any>): TimeoutID;
```

Comment:
Changed this for uniformity, to review code and functions that needed to be updated easier. There were 400+ usages of Array<> vs 40+ of []. Can change everything to [] - as you prefer